Pod/Extensions/SSO/SKYWebOAuth.m

```diff
@@ -0,0 +1,200 @@
+//
+//  SKYWebOAuth.m
+//  SKYKit
+//
+//  Copyright 2015 Oursky Ltd.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+#import "SKYWebOAuth.h"
+#import <SafariServices/SafariServices.h>
+
+@interface SKYWebOAuth () <SFSafariViewControllerDelegate>
+@end
+
+@implementation SKYWebOAuth {
+    BOOL _inProgress;
+    NSURL *_callbackURL;
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wpartial-availability"
+    SFAuthenticationSession *_authVC;
+#pragma clang diagnostic pop
+    SFSafariViewController *_safariVC;
+    UIViewController *_topVC;
```

Comment:
It is okay to use `_` for instance variable... it is just too many `_` in the code and the code cluttered : (

Reply:
I changed the private function not to start when `_`, see if it looks better? Or anything we can improve?